css/_font.scss

```diff
@@ -36,9 +36,15 @@
 .icon-navigate_before:before {
   content: "\e408";
 }
+.icon-navigate_next:before {
+  content: "\e409";
+}
 .icon-public:before {
     content: "\e80b";
 }
+.icon-restore:before {
+  content: "\e8b3";
+}
 .icon-timer:before {
     content: "\e425";
 }
```

Comment:
I used system icons for the iOS bar, because that is implemented using a native component, but this is the icon for the custom implemented android bar where I used the standard <Icon> tag. Is there another way to use system icons that I don't know of?